modules/op-release-notes-1-17.adoc

```diff
@@ -0,0 +1,187 @@
+      fetch-timeout: "1m"
+      default-url: "https://github.com/tektoncd/catalog.git"
+      default-revision: "main"
+      scm-type: "github"
+      server-url: ""
+      api-token-secret-name: ""
+      api-token-secret-key: ""
+      api-token-secret-namespace: "default"
+      default-org: ""
+      # configuration 2
+      test1.fetch-timeout: "5m"
+      test1.default-url: ""
+      test1.default-revision: "stable"
+      test1.scm-type: "github"
+      test1.server-url: "api.internal-github.com"
+      test1.api-token-secret-name: "test1-secret"
+      test1.api-token-secret-key: "token"
+      test1.api-token-secret-namespace: "test1"
+      test1.default-org: "tektoncd"
+      # configuration 3
+      test2.fetch-timeout: "10m"
+      test2.default-url: ""
+      test2.default-revision: "stable"
+      test2.scm-type: "gitlab"
+      test2.server-url: "api.internal-gitlab.com"
+      test2.api-token-secret-name: "test2-secret"
+      test2.api-token-secret-key: "pat"
+      test2.api-token-secret-namespace: "test2"
+      test2.default-org: "tektoncd-infra"
+# ...
+----
++
+.Example of using a `test1` configuration in a pipeline run
+[source,yaml]
+----
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  name: example
+spec:
+  pipelineRef:
+    resolver: git
+    params:
+    # ...
+    - name: configKey
+      value: test1
+# ...
+----
+
+* With this update, you can now monitor the running `PipelineRun` resources at the namespace, pipeline, and pipeline run level, in addition to the cluster level. You can configure the level through the `TektonConfig` custom resource by setting the `metrics.running-pipelinerun.level` parameter value to `namespace`, `pipeline`, or `pipelinerun`. The default value is empty, which sets the monitoring at cluster level.
++
+.Example of monitoring the running pipeline run at the namespace level
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+  pipeline:
+    # ...
+    metrics.running-pipelinerun.level: namespace
+# ...
+----
+
+[id="Operator-new-features-1-17_{context}"]
+=== Operator
+
+* With this update, you can run the `skopeo-copy` command with additional arguments, for example, `--all` and `--preserve-digests`. You can pass these arguments as a space-separated string, as shown in the following example:
++
+.Example command
+[source,terminal]
+----
+$ tkn task start skopeo-copy -p ARGS="--all --preserve-digests"
+----
+
+[id="pac-new-features-1-17_{context}"]
+=== {pac}
+
+* With this update, the `pipelines_as_code_running_pipelineruns_count` metric is added to show the number of running `PipelineRun` resources in {pac}. The metric can be filtered by repository or by namespace.
+
+* With this update, the `pipelines_as_code_pipelinerun_duration_seconds_sum` metric is added to show the total duration in seconds taken by all `PipelineRun` resources in {pac}. The metric can be filtered by repository or by namespace.
```

Comment:
```suggestion
* With this update, the `pipelines_as_code_pipelinerun_duration_seconds_sum` metric is added to show the total duration in seconds taken by all `PipelineRun` resources in {pac}. The metric can be filtered by repository, namespace, `PipelineRun` status, and `PipelineRun` status change reason.
```

Reply:
Hello @adellape I am just pinging you so that you are aware of this one late change coming to the PR, rest is the same, thank you!